src/modules/infobox/components/FeatureProfile.vue

```diff
@@ -306,22 +310,34 @@ export default {
                     'px'
                 // Y position of arrowhead
                 toltipEl.style.top =
-                    pos.y +
+                    y +
                     this.options.margin.top +
                     this.$refs.profilePopupContent.getBoundingClientRect().y -
                     this.$refs.profileTooltipAnchor.getBoundingClientRect().y +
                     'px'
-
+                // Tooltip text
                 toltipEl.querySelector('.distance').innerText = `${xCoord.toFixed(2)} ${
                     this.profileInfo.unitX
                 }`
                 toltipEl.querySelector('.elevation').innerText = `${yCoord.toFixed(2)} m`
+
+                // Update the position of the overlay
+                const coords = proj4(
+                    'EPSG:2056',
+                    'EPSG:3857',
+                    this.profileChart.lineString.getCoordinateAt(x / plotWidth)
+                )
+                this.currentHoverPosOverlay.setPosition(coords)
             })
             glass.on('mouseover', () => {
                 this.showTooltip = true
+                this.currentHoverPosOverlay.getElement().style.backgroundColor =
+                    this.feature.fillColor.fill
```

Comment:
As there's no unset of this property, wouldn't it be better to set that up when creating the instance, and not each time we hover?

Reply:
The thing is that the color can change if the user draws a line and then changes the color of that line. So I think that it's necessary to do it like this if we want the color of the circle to always match the color of the line.